test/OpenTelemetry.AutoInstrumentation.Tests/Configuration/SettingsTests.cs

```diff
@@ -6,108 +6,107 @@
 using OpenTelemetry.Exporter;
 using Xunit;
 
-namespace OpenTelemetry.AutoInstrumentation.Tests.Configuration
+namespace OpenTelemetry.AutoInstrumentation.Tests.Configuration;
```

Comment:
How about reformatting the whole codebase for sake of consistency?

Maybe it is only me, but I favor consistency when I am reading code. Even when new features reduces some noise. 😄

Reply:
Sure, I can do it, in this PR?

Reply:
As for me. YES 👍